docs/static/ls-ls-http.asciidoc

```diff
@@ -0,0 +1,17 @@
+//////
+This file in not currently pulled into the guide.
+It's here as a placeholder as we expand information and offerings. 
+/////
+
+
+
+
+[[ls-to-ls-http]]
+==== Logstash-to-Logstash using HTTP
+
+You can set up communication between two Logstash machines using the http output to the http input. 
+
+IMPORTANT: In its current iteration, LS-to-LS using HTTP may not solve all use cases currently addressed by the Beats-Lumberjack approach. 
+We are working on some features that will expand capabilities, and make {ls}-to-{ls} with HTTP easier to configure and use. 
+For now, <<plugins-outputs-http,http output>> to <<plugins-inputs-http,http input>> with manual configuration is the best path forward if you don't need the round-trip acknowledgment and load balancing features provided by the Lumberjack-Beats configuration. 
```

Comment:
This statement is not entirely true provided we don't really do load-balancing and both do acknowledgment, they just work differently. Let's simply point users to the plugins to configure. And once this lands, I'll work on a PR to add some basic configuration.

Reply:
This is phantom content. The changes you committed actually removed it.  It was still showing up because of the file restructuring I did (without deleting that file).

Reply:
Since then, I've rearranged the content in the source files so this file is used now--but with the content we want.